src/WordPress/Proxy.php

```diff
@@ -89,7 +89,7 @@ public function createRequest()
                 $path = API\Plugin::ENDPOINT . $_GET['proxyURL'];
             }
         } else {
-            $path = $body['proxyURL'];
+            $path = $body['proxyURL'] ?? '';
```

Comment:
similar to the above, this was a failure in CI

```
2) CF\Test\WordPress\ProxyTest::testRunHandlesPost
Trying to access array offset on value of type null

/app/src/WordPress/Proxy.php:92
/app/src/WordPress/Proxy.php:58
/app/src/Test/WordPress/ProxyTest.php:77
```

Again, it was returning `null` so instead I defaulted to an empty `proxyURL` value which _may_ still be desirable. If not, we need to rejig the test case.